cms/templates/js/license-selector.underscore

```diff
@@ -98,7 +98,7 @@
                     />
             <% } else { %>
 	        <% //<span> must come before <i> icon or else spacing gets messed up %>
-                <span class="sr">gettext("Creative Commons licensed content, with terms as follow:")&nbsp;</span><span aria-hidden="true" class="icon-cc"></span>
+                <span class="sr">Creative Commons licensed content, with terms as follow&nbsp;</span><span aria-hidden="true" class="icon-cc"></span>
```

Comment:
@sarahkf we still want to i18n this, so you want to keep the call to gettext.  You want to make sure python evaluates it.